opm/autodiff/BlackoilWellModel.hpp

```diff
@@ -195,6 +195,10 @@ namespace Opm {
                                          unsigned spaceIdx,
                                          unsigned timeIdx) const;
 
+
+            using WellInterfacePtr = std::shared_ptr<WellInterface<TypeTag> >;
+            WellInterfacePtr well(const std::string& wellName) const;
```

Comment:
Where is this function used?  Anyway, we have too many `well` related, it will be very helpful to have more detailed name like `getWell` or `getWellByName()`.

Reply:
Maybe if this function return a reference to the WellInterfacePtr, we do not need to change from `unique_ptr` to `shared_ptr`?  But I am not sure how these functions will be used, so it is just a comment.